suite-common/wallet-utils/src/__tests__/balanceUtils.test.ts

```diff
@@ -13,16 +13,15 @@ test('formatBalanceUtils', () => {
     expect(formatCoinBalance('666.666')).toEqual('666.666');
     expect(formatCoinBalance('999')).toEqual('999');
     expect(formatCoinBalance('999.01')).toEqual('999.01');
-    expect(formatCoinBalance('1000')).toEqual('1000');
-    expect(formatCoinBalance('1000.0001')).toEqual('1000.0001');
-    expect(formatCoinBalance('1000.000000001')).toEqual('1000.00');
-    expect(formatCoinBalance('2600.1')).toEqual('2600.1');
-    expect(formatCoinBalance('200000')).toEqual('200000');
-    expect(formatCoinBalance('2000000')).toEqual('2000000');
-    expect(formatCoinBalance('2900000')).toEqual('2900000');
-    expect(formatCoinBalance('100000000')).toEqual('100m');
-    expect(formatCoinBalance('100000001')).toEqual('100m+');
-    expect(formatCoinBalance('0099999999.999999999999')).toEqual('99999999.9…');
+    expect(formatCoinBalance('1000')).toEqual('1,000');
+    expect(formatCoinBalance('1000.0001')).toEqual('1,000.0001');
+    expect(formatCoinBalance('1000.000000001')).toEqual('1,000.00');
+    expect(formatCoinBalance('2600.1')).toEqual('2,600.1');
+    expect(formatCoinBalance('200000')).toEqual('200,000');
+    expect(formatCoinBalance('2000000')).toEqual('2,000,000');
+    expect(formatCoinBalance('2900000')).toEqual('2,900,000');
+    expect(formatCoinBalance('0099999999.999999999999')).toEqual('99,999,999.9…');
     expect(formatCoinBalance('0.12345678')).toEqual('0.12345678');
     expect(formatCoinBalance('10.12345678')).toEqual('10.1234567…');
+    expect(formatCoinBalance('10.12345678', 'cs')).toEqual('10,1234567…');
```

Comment:
I would suggest to add more cases for all the locales available in Suite

Reply:
97bf0112f9b74c51f4c29d88982cf9e0e3388d9c